src/ble/BleLayer.cpp

```diff
@@ -789,6 +790,18 @@ void BleLayer::OnConnectionError(void * appState, CHIP_ERROR err)
     layer->mBleTransport->OnBleConnectionError(err);
 }
 
+// BLE Flag to wait for packet transmission confirmation
+static bool BleTerminating = false;
```

Comment:
Why using a global if `BleLayer` is a class that can have regular members? Also, how about a name such as `mTerminateOnPacketTxComplete` or something that better describes the behavior?

Reply:
Changed to use regular member.